EarlGreyTests/EarlGrey.swift

```diff
@@ -0,0 +1,118 @@
+//
```

Comment:
Is it cool if we move these to ThirdParty/EarlGreyTests or UITests/EarlGreyTests. Too many top level folders lol.